### PR TITLE
[server][dvc] tweak logging for server

### PR DESCRIPTION
**Pull request by adamxchen**
## Summary, imperative, start upper case, don't end with a period

When the host has a few hundreds or even thousands of stores, this log line spam the log file easily.

## How was this PR tested?

n/a

## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

**Review by KaiSernLim — APPROVED** [review]

File: clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStorageEngineFactory.java [hunk 1]
```diff
@@ -280,7 +280,7 @@ public synchronized Set<String> getPersistedStoreNames() {
     File databaseDir = new File(rocksDBPath);
     if (databaseDir.exists() && databaseDir.isDirectory()) {
       String[] storeDirs = databaseDir.list();
-      LOGGER.info("Found the following RocksDB databases: {}", Arrays.toString(storeDirs));
```
Comment (majisourav99) [inline clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStorageEngineFactory.java:283]: may be this one by default could be helpful to find how many DB the server found ?

File: internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java [hunk 2]
```diff
@@ -106,10 +106,10 @@ private static <T> T createFactory(
     String className;
     if (properties.containsKey(configKey)) {
       className = properties.getString(configKey);
-      LOGGER.info("Creating pub-sub {} adapter factory instance for class: {}", factoryType, className);
```
Comment (haoxu07) [inline internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java:109]: Seems this is not for rocksdb. Is this too much in production?

File: clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStorageEngineFactory.java (hunk 1 above)
Comment (adamxchen) [reply clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStorageEngineFactory.java:283]: I'm not sure about this.. How are we going to use this information tho? The reason why I wanted to remove this is when a single host have too many replica, this log line would be huge and is hard to parse.

Comment (adamxchen) [reply clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStorageEngineFactory.java:283]: The good thing is it only send out once for startup. Let me know what you think. We can keep it if you find it can be helpful for debugging.

File: internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java (hunk 2 above)
Comment (adamxchen) [reply internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java:109]: Updated the title. Just some logging change from minor issues I've noticed during oncall. When a same pubsub system is used, say Kafka in this cluster, this log would repeat multiple times according to the number of stores it have. For example, if a host has 1000 replicas and it uses Kafka, then we would have 
```
 [PubSubClientsFactory] [main] Creating pub-sub PRODUCER adapter factory instance with default class: com.linkedin.venice.pubsub.adapter.kafka.producer.ApacheKafkaProducerAdapterFactory
 [PubSubClientsFactory] [main] Creating pub-sub CONSUMER adapter factory instance with default class: com.linkedin.venice.pubsub.adapter.kafka.consumer.ApacheKafkaConsumerAdapterFactory
 [PubSubClientsFactory] [main] Creating pub-sub ADMIN adapter factory instance with default class: com.linkedin.venice.pubsub.adapter.kafka.admin.ApacheKafkaAdminAdapterFactory
```
Repeating 1000 times, which isn't really useful IMO.

Comment (adamxchen) [reply internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java:109]: If you believe there's a good value to keep it, I can make this a `DEBUG`

Comment (haoxu07) [reply internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java:109]: Ideally, it should not be that many occurrences, as this is a factory class.

Comment (sushantmane) [reply internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java:109]: >Repeating 1000 times, which isn't really useful IMO.

That's not good. Do you remember if you saw this on controller or servers?

Comment (sushantmane) [reply internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java:109]: I checked the code and it should be one log line per client per host

Comment (adamxchen) [reply internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java:109]: > (For producer's it could be equal to VW factory instances per host)

I recall @haoxu07 did some search on pre-production env. He did observe this line of log is spamming due to the number of `VeniceServerConfig`. @haoxu07  can you please share your findings? I can make the log DEBUG if it has some values

Comment (sushantmane) [reply internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubClientsFactory.java:109]: It does help in figuring out what type of clients we are using but we can infer that using other means as well. So it should be ok to make it debug I guess
